pkg/bpf/bpf_kmesh_workload.go

```diff
@@ -385,8 +396,25 @@ func (sm *BpfSendMsgWorkload) loadKmeshSendmsgObjects() (*ebpf.CollectionSpec, e
 		return nil, err
 	}
 
+	// sendmsg ebpf prog is mounted on sockmap and processed for each socket.
+	// It has the following characteristics:
+	// 1. Multiple sk_msg ebpf prog can exist at the same time
+	// 2. If the old sk_msg ebpf program is not pinned, it will wait until all
+	// sockets on the old sk_msg ebpf prog are disconnected before automatically detaching.
+	// Therefore, the following methods are used to achieve seamless replacement
+	// 1) loading new sk_msg prog
+	// 2) unpin old psk_msg rog
+	// 3) pin new sk_msg prog
+	// 4) attach new sk_msg prog
```

Comment:
Describe the service logic changes after the preceding operations are performed.